tests/features/consumer/searchDocumentReference-failure.feature

```diff
@@ -151,10 +147,10 @@ Feature: Consumer - searchDocumentReference - Failure Scenarios
       | url         | https://example.org/my-doc.pdf    |
       | custodian   | 8FW23                             |
       | author      | 8FW23                             |
-    When consumer 'RX898' searches for DocumentReferences with parameters:
+    When consumer 'Z26' searches for DocumentReferences with parameters:
       | parameter | value      |
       | subject   | 9278693472 |
-      | type      | 736253002  |
+      | type      | 123        |
```

Comment:
The consumer wouldn't have permissions for `736253002` anyways? Why make up a `123` pointer type?